spec/fixtures/pdf_fill/10-10CG/kitchen_sink.json

```diff
@@ -1,103 +1,95 @@
 {
-  "veteran": {
```

Comment:
is this just changing the tabs from 2 spaces to 4?  looking in the rest of this directory, it looks like `overflow.json` is 4 but `simple.json` is 2.  Looking around, it looks like the other pdf_fills are using 2 space tabs.  So I think this should be using 2 spaces.